tern/analyze/docker/dockerfile.py

```diff
@@ -254,6 +254,13 @@ def find_git_info(line, dockerfile_path):
     logger.debug('Parsed src_path is %s', src_path)
     # get the git project info
     comment_line = common.check_git_src(dockerfile_path)
+    # get the git project link
+    url_list = common.get_git_url(dockerfile_path)
+    if url_list:
+        comment_url = ''
+        for url in url_list:
+            comment_url += url + ' '
+        comment_line += ', project url: ' + comment_url
```

Comment:
small nit: change `project url` to `project url(s)` since there could be more than one.